### DISCUSSION TITLE
DateCalendar/DatePicker with range option is not available in Medusa 2.0

### DISCUSSION THREAD

**Discussion by radoslaw-sz** [medusajs/medusa#10689] [Feature Requests]
Hi,

I used to use DateCalendar in one of my plugins with the mode `range` - thanks to that it was possible to mark the range of dates. Unfortunately it looks like in Medusa 2.0 together with new Medusa UI versions it is no longer available.

Do you have plan to introduce it again?

**Comment by BlazejBatko** [comment]
any updates on that?
